doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -529,6 +529,10 @@ Displaying information about notations
     :cmd:`Locate` for information on the definitions and scopes associated with
     a notation.
 
+.. cmd:: Print Keywords
+
+   Prints the current reserved keywords, one per line.
```

Comment:
```suggestion
   Prints the current keywords, one per line.  Keywords cannot be used as identifiers.
```

The current description doesn't use "reserved"; it's simpler just to state the limitation.
And hyperlink to `basic.rst` "Keywords" title.

Maybe state what actions can change the list of keywords (eg loading a plugin).

Also would be nice to mention when the command is helpful (use cases, e.g. when you're picking an identifier).